src/components/ebay-skeleton-loader/index.marko

```diff
@@ -0,0 +1,48 @@
+import { processHtmlAttributes } from "../../common/html-attributes";
+static var ignoredAttributes = [
+    "a11yText",
+    "class",
+    "size",
+    "type",
+    "multiline",
+    "count",
+    "columns",
+    "isVisible",
+];
+static var validTypes = ["avatar", "button", "text", "form", "image", "tile"];
+static var validSizes = ["large", "small"];
+$ var size = validSizes.includes(input.size) ? input.size : "default";
+$ var type = validTypes.includes(input.type) ? input.type : "button";
+$ var typeClass = type === "button" ? "skeleton--btn" : `skeleton--${type}`;
+$ var skeletonClass = (
+    size !== "default"
+        ? `${typeClass}-${size}`
+        : type === "text"
+        ? `${typeClass}-small`
+        : typeClass
+);
+
+<if(typeof input.isVisible === "undefined" ? true : input.isVisible)>
+    <div
+        ...processHtmlAttributes(input, ignoredAttributes)
+        class=[
+            type !== "tile" ? `skeleton ${skeletonClass}` : typeClass,
+            input.multiline && type === "text" && "skeleton--text-multiline",
+            input.class,
+        ]
+        role="img"
+        aria-label=input.a11yText || "Loading"
+    >
+        <if(type === "tile")>
```

Comment:
This is a lot of complex logic for the `tile` case, maybe it's a reason to move `ebay-skeleton-tile` into a separate component which has consistent logic (not sure about this comment, just a thought)

Reply:
I think the right way is to make this component callable from others. 
Lets leave it as is for now. In future we should have button or tile having a "loading" option which would swap it to a skeleton component.